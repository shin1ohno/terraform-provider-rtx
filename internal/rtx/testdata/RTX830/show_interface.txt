LAN1: UP
  IP Address: 192.168.1.254/24
  MAC Address: 00:A0:DE:12:34:56
LAN2: DOWN
  IP Address: Not configured
  MAC Address: 00:A0:DE:12:34:57
WAN1: UP
  IP Address: 203.0.113.1/30
  MAC Address: 00:A0:DE:12:34:58
PP1: UP
  IP Address: PPPoE
VLAN10: UP
  IP Address: 10.10.10.1/24
  MAC Address: 00:A0:DE:12:34:59