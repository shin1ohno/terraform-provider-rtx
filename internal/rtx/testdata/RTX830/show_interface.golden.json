[
  {
    "name": "LAN1",
    "kind": "lan",
    "admin_up": true,
    "link_up": true,
    "mac": "00:A0:DE:12:34:56",
    "ipv4": "192.168.1.254/24"
  },
  {
    "name": "LAN2",
    "kind": "lan",
    "admin_up": false,
    "link_up": false,
    "mac": "00:A0:DE:12:34:57"
  },
  {
    "name": "WAN1",
    "kind": "wan",
    "admin_up": true,
    "link_up": true,
    "mac": "00:A0:DE:12:34:58",
    "ipv4": "203.0.113.1/30"
  },
  {
    "name": "PP1",
    "kind": "pp",
    "admin_up": true,
    "link_up": true
  },
  {
    "name": "VLAN10",
    "kind": "vlan",
    "admin_up": true,
    "link_up": true,
    "mac": "00:A0:DE:12:34:59",
    "ipv4": "10.10.10.1/24"
  }
]